docs/en/verifier.md

```
@@ -516,34 +506,38 @@ createAuthzRequest(
 **Return value**:
-- Returns an `AuthorizationRequest` object ([AuthorizationRequest](#AuthorizationRequest)). This object takes one of the following forms:
-
-  - **Request URI format** (when `isRequestUri = true`):
-  ```typescript
-  {
-    client_id: string,
-    request_uri: string
-  }
-  ```
+- Returns `{ request: AuthorizationRequest, transactionId: string }`.
+  - `request` ([AuthorizationRequest](#AuthorizationRequest)): takes one of the following forms:
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)